math/binarymath/checkisnumberpoweroftwo_test.go

```diff
@@ -0,0 +1,92 @@
+// checkisnumberpoweroftwo_test.go
+// description: Test for Is the number a power of two
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see checkisnumberpoweroftwo.go
+
+package binarymath
+
+import (
+	"math"
+	"testing"
+)
+
+func getTestsForPowerOfTwo() []struct {
+	name    string
+	a       int
+	missing bool
+} {
+	var tests = []struct {
+		name    string
+		a       int
+		missing bool
+	}{
+		{"64 is power of two? - YES", 64, true},
```

Comment:
"Is 64 a power of 2?" would be grammatically better :)

Reply:
Done